packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1654,6 +1684,7 @@ export class ContainerRuntime extends EventEmitter
     private async generateSummary(
         fullTree: boolean = false,
         safe: boolean = false,
```

Comment:
Unrelated to your changes: I'd change meaning / semantics of safe.
I'd separate fullTree & refreshPreviousAck as two distinct and non-overlapping properties.
I.e. where safe == true today, I'd pass fullTree = true & refreshPreviousAck = true instead.

Reply:
Renamed `fullTree` to `cannotReuseHandle` and distinguished it from `refreshPreviousAck`.